tweb/src/main/frontend/src/scss/fundrequest/_form.scss

```diff
@@ -162,3 +162,22 @@
     }
 }
 
+.placeholder-gray::-webkit-input-placeholder {
```

Comment:
make it one rule 

.placeholder-gray::-webkit-input-placeholder,
.placeholder-gray:-moz-placeholder,
.placeholder-gray::-moz-placeholder,
..... {
     color: $gray-300;
}